fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1365,7 +1366,39 @@ public PlanFragment visitPhysicalIcebergMetadataScan(OptExpression optExpression
 
             IcebergMetadataScanNode metadataScanNode =
                     new IcebergMetadataScanNode(context.getNextNodeId(), tupleDescriptor,
-                            "IcebergMetadataScanNode");
+                            "IcebergMetadataScanNode", node.getTemporalClause());
+            try {
+                ScalarOperatorToExpr.FormatterContext formatterContext =
+                        new ScalarOperatorToExpr.FormatterContext(context.getColRefToExpr());
+                List<ScalarOperator> predicates = Utils.extractConjuncts(node.getPredicate());
```

Comment:
this predicate is "predicate = serialized string"?

Reply:
yes